src/worker.js

```diff
@@ -7,20 +7,17 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
-export default (app, config, onServer, workerId) => {
-  let server;
-
-  // ===== Middleware =========================================================
+export const attachMiddleware = (app, config) => {
```

Comment:
Not really related, but maybe you know what's going on.
Atom complains about some eslint preset, although I did full `npm install` in the repo, so all the dependencies should be in place.

![atom eslint error](http://ia.gs/gZCs/Image%202016-06-27%20at%206.57.47%20PM.png)

And when I run eslint in the terminal it's pretty happy.